### PR TITLE
bump version and sync readme

### PR DESCRIPTION
**Pull request by bmorelli25**
Fixes documentation bug, syncs readme, and bumps version.

* Also for https://github.com/elastic/integrations/pull/5188

### CODE REVIEW

File: packages/system_audit/changelog.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+- version: "1.0.1"
+  changes:
+    - description: Fix documentation bug
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/4862
```
Comment (bmorelli25) [inline packages/system_audit/changelog.yml:5]:
```suggestion
      link: https://github.com/elastic/integrations/pull/5203
```

**Comment by jamiehynds** [comment]
LGTM. Ok with you @ShourieG?

**Review by ShourieG — APPROVED** [review]
LGTM

**Pull request merged**